e2e/scenario_test.go

```diff
@@ -957,6 +957,35 @@ func Test_Ubuntu2204_GPUNoDriver(t *testing.T) {
 	})
 }
 
+func Test_Ubuntu2204_GPUNoDriver_Scriptless(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "Tests that a GPU-enabled node using the Ubuntu 2204 VHD opting for skipping gpu driver installation can be properly bootstrapped",
+		Tags: Tags{
+			GPU: true,
+		},
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			AKSNodeConfigMutator: func(config *aksnodeconfigv1.Configuration) {
+				config.VmSize = "Standard_NC6s_v3"
+				config.GpuConfig.ConfigGpuDriver = true
+				config.GpuConfig.GpuDevicePlugin = false
+				config.GpuConfig.EnableNvidia = to.Ptr(true)
+			},
+			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {
+				vmss.Tags = map[string]*string{
+					// deliberately case mismatched to agentbaker logic to check case insensitivity
+					"SkipGPUDriverInstall": to.Ptr("true"),
```

Comment:
do use VMSS tags for provisioning logic?

Reply:
Other tests are not using. I think we don't?

Reply:
If it's not required, can you please remove it?

It feels like there is hidden logic behind it otherwise.

Reply:
oh it's needed because the previous test `Test_Ubuntu2204_GPUNoDriver` also has it. And this tag was checked in cse_main.sh with this logic 
```
export -f should_skip_nvidia_drivers
skip_nvidia_driver_install=$(retrycmd_if_failure_no_stats 10 1 10 bash -cx should_skip_nvidia_drivers)
```
Otherwise the test would fail.